addons/radar/XEH_preInit.sqf

```diff
@@ -524,5 +539,37 @@ if !(hasInterface) exitWith {};
     true
 }] call CBA_fnc_addKeybind;
 
+if (_tfar) then {
+    ["TFAR_event_OnSpeak", {
+        if !(GVAR(showSpeaking)) exitWith {};
+        params ["_unit", "_isSpeaking"];
+        _unit setVariable [QGVAR(isSpeaking), _isSpeaking, true];
```

Comment:
where is that varible set to "false" for the TFAR case?

Reply:
As soon as you stop talking it sets back to false. Unsure if it set to false upon unit death.

Reply:
:facepalm: oh, I see it now. right. :+1: